packages/legend-application-studio/src/stores/editor-state/element-editor-state/service/ServiceExecutionState.ts

```diff
@@ -409,15 +416,18 @@ export class SingleExecutionContextState extends ServiceExecutionContextState {
   }
 
   setMapping(value: Mapping): void {
+    this.executionContext.mapping =
```

Comment:
can just do 
```
    pureSingleExecution_setMapping(
      this.executionContext1,
      PackageableElementExplicitReference.create(value),
      this.executionState.editorStore.changeDetectionState.observerContext,
    );

```

Reply:
I had to explicit set it, otherwise the runtime value was not getting populated readily in the UI